src/modules/navigator/rtl_params.c

```diff
@@ -173,3 +173,15 @@ PARAM_DEFINE_INT32(RTL_PLD_MD, 0);
  * @group Return Mode
  */
 PARAM_DEFINE_FLOAT(RTL_LOITER_RAD, 50.0f);
+
+/**
+ * RTL heading mode
+ *
+ * Defines the heading behavior during RTL
+ *
+ * @value 0 Towards next waypoint.
+ * @value 1 Heading matches destination.
+ * @value 2 Use current heading.
+ * @group Return To Land
```

Comment:
Shouldn't it be `@group Return Mode`?

Reply:
It looks like `RTL_PLD_MD` was also labeled `Return To Land` and must have been what I copied/pasted from.  I'll standardize so that all are `@group Return To Land` and rebase against current master.  Thanks for reviewing @sfuhrer !